cfe_internal/enterprise/federation/federation.cf

```diff
@@ -231,26 +250,43 @@ bundle agent postgres_config
 {
   vars:
     am_superhub::
-      "c[shared_buffers]" string => "1GB";
-      "c[max_locks_per_transaction]" string => "4000";
-
+      "c[shared_buffers]"
```

Comment:
Do we have any issues related to these config value settings? Seems some promisees might ge good for reference.

This is an ideal spot to hook in for automatically configuring postgres based on the information we have about how it's functioning/hardware available.

Reply:
created one: https://tracker.mender.io/browse/ENT-4614